xdg-portal/Makefile.am

```
@@ -0,0 +1,8 @@
+NULL =
+
+xdgportaldir = $(datadir)/xdg-desktop-portal
+xdgportal_DATA = mate-portals.conf
```

If you changed this to

```suggestion
dist_xdgportal_DATA = mate-portals.conf
```

then you wouldn't need to put it in `EXTRA_DIST`, if I remember correctly how Autotools works.

---

Yeap that works, `make distcheck` creates a proper tarball with `xdgportaldir`